lib/rouge/demos/ecl

```diff
@@ -0,0 +1,59 @@
+// inline comment
+
+/**************************
+This is a multiline comment
+***************/
+
+import std.math.job;
+IMPORT Str;
+
+// Types
+R1 := RECORD
+    integer1  F1;
+    UNSIGNED4 F2;
+    STRING100 F3;
+    REAL4     R1;
+    REAL8     R2;
+    REAL9     R2; // This should not match a type
+    boolean   B1;
+    decimal23 D1;
+    udecimal3 D2;
+END;
```

Comment:
@dehilsterlexis can you add a nested RECORD as well?

Also add a MODULE,  MODULEMACRO and EMBED samples?

Reply:
(I don't expect the contents of and EMBED to be colored correctly)

Reply:
This was more of a placeholder for submitting this in a timely manner. I am currently working on a full test file which I will submit.

Reply:
This is actually overly long for the demo file. That file is only intended to be a short example of the language. Anything more substantial should be in the visual sample. Could you cut this down a bit?

Reply:
@GordonSmith you are right about the EMBED etc. Fixing now.

Reply:
@pyrmont yea. I got this wrong. I simply copied my test file. I will replace this with our short "standard" example.